config.py

```diff
@@ -380,7 +380,6 @@ def setup_extension_modules(self):
         if self.debug_mode:
             for k in self.compiler_flags:
                 self.compiler_flags[k].append('-g')
-                self.compiler_flags[k].append('-fno-omit-frame-pointer')
```

Comment:
It's not a nvcc flag, it's a compiler flag to help with debugging large functions. I read about it there :
[](https://stackoverflow.com/questions/14666665/trying-to-understand-gcc-option-fomit-frame-pointer
)and decided to add it.

Reply:
Ah, OK, thanks for catching that. It seems that nvcc doesn't allow this option, so, I've changed the script to now only apply it if nvcc isn't used.